fog/view/server/src/metrics.rs

```diff
@@ -0,0 +1,35 @@
+use lazy_static::lazy_static;
+use prometheus::{
+    histogram_opts, register_histogram, register_histogram_vec, register_int_counter,
+    register_int_gauge, Histogram, HistogramVec, IntCounter, IntGauge,
+};
+
+// Initialize global metrics
+lazy_static! {
+    pub static ref QUERY_REQUESTS: HistogramVec = register_histogram_vec!(
```

Comment:
Similar to the comment in the ledger server, I think renaming to `STORE_QUERY_REQUESTS` and `ROUTER_QUERY_REQUESTS` would make this a bit more intuitive.